code/modules/projectiles/guns/pistols.dm

```diff
@@ -258,7 +264,7 @@
 	item_state = "g22"
 	caliber = CALIBER_9X19_TRANQUILIZER //codex
 	max_shells = 12 //codex
-	default_ammo_type =/obj/item/ammo_magazine/pistol/g22tranq
+	default_ammo_type = /obj/item/ammo_magazine/pistol/g22tranq
```

Comment:
missing an allowed ammo type there

Reply:
it inherits, unless its only allowed the tranq mags?

Reply:
is the parent allowing /obj/item/ammo_magazine/pistol/g22tranq ?

Reply:
yeah